logoapp/upload.php

```diff
@@ -0,0 +1,10 @@
+<?php
+echo '<pre>';
+echo $_POST["image"];
+
+if(move_uploaded_file($_POST["image"], "upload/test.jpg")){
+  echo "<br>uploaded";
+  exit();
+}else{
+  echo "<br>upload error";
+}
```

Comment:
__[code_sniffer]__
* Expected 1 newline at end of file; 0 found

<!-- Comment by [SideCI](https://sideci.com) -->